Makefile

```diff
@@ -432,10 +432,12 @@ istioctl-test: istioctl
 	go test -p 1 ${T} ./istioctl/...
 
 .PHONY: mixer-test
-MIXER_TEST_T ?= ${T} ${GOTEST_PARALLEL}
+MIXER_TEST_T ?= ${T} -count 1 -parallel 1 -p 1 -cpu 1
```

Comment:
I am wondering if we should break out these tests into a separate target and disable parallelism only for these instead? WDYT?

Reply:
The changes to the Makefile have been reverted. The problem is caused by the hot restart feature of Envoy.